vendor/k8s.io/kubernetes/pkg/proxy/iptables/proxier_openshift.go

```diff
@@ -5,7 +5,7 @@ package iptables
 import "k8s.io/kubernetes/pkg/util/async"
 
 func (p *Proxier) SyncProxyRules() {
-	p.maybeSyncProxyRules()
+	p.forceSyncProxyRules()
```

Comment:
In the very last commit on the branch you made this change while untangling other commits, but this is wrong; it should stay as `maybeSyncProxyRules` in this file

Reply:
done